### ISSUE TITLE
Tern lint tries to guess field type by name if used  unknown identifier, it looks confusing.

### ISSUE THREAD

**Issue by dmitriiKovchin** [angelozerr/tern-lint#60]
First warning is expected, but why we continue validation even when we understood that identifier is unknown. 

![screenshot from 2015-10-16 15 23 02](https://cloud.githubusercontent.com/assets/13972393/10554666/834b0f02-741a-11e5-8b53-3548ca99daee.png)

**Comment by angelozerr** [comment]
Yes you are right. But I don't know if it's easy to do. Any contribution are welcome.
